docs/software/python.md

```diff
@@ -1,16 +1,10 @@
 # Python and R with Anaconda
 
-To support the diverse _python_ and _R_ workflows and high levels of 
-customization Research Computing users require, 
-[Anaconda](http://anaconda.com) is installed on the CURC system. Anaconda 
-uses the _conda_ package manager to easily install software and associated 
-packages, and supports _python_, _R_, and many other applications. The 
-following documentation describes how to activate the CURC Anaconda 
-distribution and our default environments, as well as how to create and 
-activate your own custom Anaconda environments. For more information on utilizing [Mamba](https://mamba.readthedocs.io/) (an alternative package manager) please see the section below. Additional documentation 
-on [CURC OpenOnDemand](../gateways/OnDemand.md) is available for users 
-desiring to 
-interact with their custom environments via [Jupyter notebooks](https://jupyter.org). 
+To support diverse _Python_ and _R_ workflows, Research Computing users can utilize  
+[Anaconda](http://anaconda.com). Anaconda 
+provides the _conda_ package manager, which allows for easy installation of software and associated 
+packages. The _conda_ package manager provides support for _Python_, _R_, and many other application stacks. The 
+following documentation describes how to activate the CURC Anaconda distribution and our default environments, as well as how to create and activate your own custom Anaconda environments. For more information on utilizing [Mamba](https://mamba.readthedocs.io/) (an alternative package manager) please see the section [Mamba Package Manger](#mamba-package-manager) below. Additional documentation on [CURC OpenOnDemand](../gateways/OnDemand.md) is available for users desiring to interact with their custom environments via [Jupyter notebooks](https://jupyter.org). 
```

Comment:
To support diverse _Python_ and _R_ workflows, Research Computing users can utilize  
[Anaconda](http://anaconda.com). Anaconda 
provides the _conda_ package manager, which allows for easy installation of software and associated 
packages. The _conda_ package manager provides support for _Python_, _R_, and many other application stacks. CURC also supports Mamba (link), an alternative package manager that allows parallel downloading of repository data and package files using multi-threading. 
new line
The following sections describe how to activate the CURC Anaconda distribution and our default environments, how to create and activate your own custom Anaconda environments, and basic Mamba usage instruction.